docs/05-concepts/02-models.md

```diff
@@ -159,3 +159,51 @@ extension MyExtension on MyClass {
 |[**fields (index)**](database/indexing)|List the fields to create the indexes on.                                                |✅|||
 |[**type (index)**](database/indexing)|The type of index to create.                                                               |✅|||
 |[**unique**](database/indexing)|Boolean flag to make the entries unique in the database.                                         |✅|||
+
+## Default Values
+
+Serverpod supports defining default values for fields in your models. These defaults can be specified using the following keywords:
+
+#### Keywords
+
+- **default**: Sets the default value for both the model and the database.
+- **defaultModel**: Sets the default value for the model side.
+- **defaultDatabase**: Sets the default value for the database side.
```

Comment:
Suggestion: Add these keywords to the table above that lists all keywords supported for a model.

Reply:
If these are listed in the list above then we don't need to repeat that information here and this section can be removed.